includes/class-article-settings.php

```diff
@@ -60,6 +61,36 @@ public function register_meta_boxes() {
 			'advanced',
 			'default'
 		);
+
+		add_meta_box(
+			'republication-tracker-tool-hide-widget',
+			esc_html__( 'Hide Republication Widget', 'republication-tracker-tool' ),
+			array( $this, 'render_hide_widget_metabox' ),
+			array( 'post', 'page' ),
+			'side',
+			'default'
+		);
```

Comment:
Should we set the seventh parameter of add_meta_box here to

```php
array(
    '__block_editor_compatible_meta_box' => true,
)
```

?

It's not explicitly required, but it may help forwards compatibility. https://developer.wordpress.org/block-editor/developers/backward-compatibility/meta-box/#testing-converting-and-maintaining-existing-meta-boxes

Do we want this checkbox's value to update on the gutenberg save action, rather than on page refresh?

Reply:
@benlk The new parameter is in https://github.com/INN/republication-tracker-tool/pull/54/commits/1002f263b842a31385d6723e6b7f9bb52182f7df.